core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -1394,20 +1394,20 @@ public static String printLine(String a, int num) {
    * Below method will be used to get the list of segment in
    * comma separated string format
    *
-   * @param segmentList
+   * @param strings
    * @return comma separated segment string
    */
-  public static String getSegmentString(List<String> segmentList) {
-    if (segmentList.isEmpty()) {
+  public static String convertToString(List<String> strings) {
```

Comment:
please modify input param to a more readable name and update the comment for this function

Reply:
ok